### ISSUE TITLE
describe repositories is not pagable

### ISSUE THREAD

**Issue by shurrman** [awslabs/ecr-cleanup-lambda#19]
Hi!

I get this error when trying to run the function

``` python main.py
Discovering images in us-east-1
Traceback (most recent call last):
  File "main.py", line 195, in <module>
    handler(request, None)
  File "main.py", line 44, in handler
    discover_delete_images(endpoint)
  File "main.py", line 53, in discover_delete_images
    describerepo_paginator = ecr_client.get_paginator('describe_repositories')
  File "/usr/local/lib/python2.7/dist-packages/botocore/client.py", line 554, in get_paginator
    raise OperationNotPageableError(operation_name=operation_name)
botocore.exceptions.OperationNotPageableError: Operation cannot be paginated: describe_repositories
```

**Comment by anshrma** [comment]
Hi - Can you please print the version of botocore that you are using ?
describe_repositories paginator was released with release 1.4.69 https://github.com/boto/botocore/commit/378f9bdc9a6338aa6531a5c43f08a42556a7ba21

Hence you should be above that version.
I tried with aws-cli/1.11.135 Python/3.6.1 Darwin/15.6.0 botocore/1.6.2 and i dont get this error and continue to work fine.

**Comment by shurrman** [comment]
@anshrma 

Thanks for the hint, I was running it with python 2.7 

Probably this should be changed in the readme ;-) `--runtime python2.7`
